include/erlcloud_aws.hrl

```diff
@@ -20,7 +20,9 @@
 
 -record(aws_config, {
           as_host="autoscaling.amazonaws.com"::string(),
+          ec2_protocol="https"::string(),
           ec2_host="ec2.amazonaws.com"::string(),
+          ec2_port=443::non_neg_integer(),
```

Comment:
to be backward compatible this should default to `undefined`
otherwise this makes ALL EC2 calls do `:443/` and not `/`
https://github.com/erlcloud/erlcloud/blob/master/src/erlcloud_aws.erl#L246-L247

granted it does not change much for your app but can affect other mecking lhttpC calls

https://docs.aws.amazon.com/general/latest/gr/rande.html#regional-endpoints

Reply:
Handled in eff118f